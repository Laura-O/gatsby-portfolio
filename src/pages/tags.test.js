import React from 'react';
import { shallow } from 'enzyme';
import { sitePropType, tagsPropType } from '../../test/fixtures';
import TagsPage from './tags';

test('rendering', () => {
  const props = {
    data: {
      site: sitePropType,
      allMarkdownRemark: {
        group: [tagsPropType],
      },
    },
  };
  const wrapper = shallow(<TagsPage {...props} />);
  expect(wrapper).toMatchSnapshot();
});
