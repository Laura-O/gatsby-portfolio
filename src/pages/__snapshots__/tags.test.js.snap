// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TagsPage
    data={
      Object {
        "allMarkdownRemark": Object {
          "group": Array [
            Object {
              "fieldValue": "test1",
              "totalCount": 1,
            },
          ],
        },
        "site": Object {
          "siteMetadata": Object {
            "title": "title",
          },
        },
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
          title="title"
        />,
        <div>
          <h1>
            Tags
          </h1>
          <ul>
            <li>
              <Unknown
                to="/tags/test-1/"
              >
                test1
                 (
                1
                )
              </Unknown>
            </li>
          </ul>
        </div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "defer": true,
          "encodeSpecialCharacters": true,
          "title": "title",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1>
              Tags
            </h1>,
            <ul>
              <li>
                <Unknown
                  to="/tags/test-1/"
                >
                  test1
                   (
                  1
                  )
                </Unknown>
              </li>
            </ul>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Tags",
            },
            "ref": null,
            "rendered": "Tags",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li>
                  <Unknown
                    to="/tags/test-1/"
                  >
                    test1
                     (
                    1
                    )
                  </Unknown>
                </li>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": "test1",
                "nodeType": "host",
                "props": Object {
                  "children": <Unknown
                    to="/tags/test-1/"
                  >
                    test1
                     (
                    1
                    )
                  </Unknown>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": Array [
                      "test1",
                      " (",
                      1,
                      ")",
                    ],
                    "to": "/tags/test-1/",
                  },
                  "ref": null,
                  "rendered": Array [
                    "test1",
                    " (",
                    1,
                    ")",
                  ],
                  "type": [Function],
                },
                "type": "li",
              },
            ],
            "type": "ul",
          },
        ],
        "type": "div",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
            title="title"
          />,
          <div>
            <h1>
              Tags
            </h1>
            <ul>
              <li>
                <Unknown
                  to="/tags/test-1/"
                >
                  test1
                   (
                  1
                  )
                </Unknown>
              </li>
            </ul>
          </div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "defer": true,
            "encodeSpecialCharacters": true,
            "title": "title",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1>
                Tags
              </h1>,
              <ul>
                <li>
                  <Unknown
                    to="/tags/test-1/"
                  >
                    test1
                     (
                    1
                    )
                  </Unknown>
                </li>
              </ul>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Tags",
              },
              "ref": null,
              "rendered": "Tags",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li>
                    <Unknown
                      to="/tags/test-1/"
                    >
                      test1
                       (
                      1
                      )
                    </Unknown>
                  </li>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": "test1",
                  "nodeType": "host",
                  "props": Object {
                    "children": <Unknown
                      to="/tags/test-1/"
                    >
                      test1
                       (
                      1
                      )
                    </Unknown>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "children": Array [
                        "test1",
                        " (",
                        1,
                        ")",
                      ],
                      "to": "/tags/test-1/",
                    },
                    "ref": null,
                    "rendered": Array [
                      "test1",
                      " (",
                      1,
                      ")",
                    ],
                    "type": [Function],
                  },
                  "type": "li",
                },
              ],
              "type": "ul",
            },
          ],
          "type": "div",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
